spec/appcenter_upload_spec.rb

```diff
@@ -272,6 +272,53 @@ def stub_add_to_group(status)
       end").runner.execute(:test)
     end
 
+    it "should notify users" do
+      stub_check_app(200)
+      stub_create_release_upload(200)
+      stub_upload_build(200)
+      stub_update_release_upload(200, 'committed')
+      stub_add_to_group(200)
+      stub_get_release(200)
+
+      values = Fastlane::FastFile.new.parse("lane :test do
+        Actions.lane_context[SharedValues::FL_CHANGELOG] = 'autogenerated changelog'
```

Comment:
Is this line needed in the context of this spec?

Reply:
True, I removed it.